buildSrc/src/main/kotlin/Ext.kt

```diff
@@ -1,5 +1,5 @@
 object Ext {
   val compileSdkVersion = 29
   val ndkAbiFilters = listOf("x86", "x86_64", "armeabi-v7a", "arm64-v8a")
-  val kotlinPluginId = "app.cash.zipline.ktbridge.plugin"
+  val kotlinPluginId = "app.cash.zipline.kotlinir"
```

Comment:
Don't think using "ir" provides much value. In 6 months there will only be IR anyway. Can we do just .kotlin?

Reply:
I wanted `ir` to convey ASTs and code rewriting more than the IR API in particular. Maybe just saying `kotlin` felt unclear, and `kotlincompilerplugin` too verbose. I’ll go with `kotlin` unless you have other recommendations!